cluster-autoscaler/cloudprovider/externalgrpc/examples/external-grpc-cloud-provider-service/wrapper/wrapper.go

```diff
@@ -125,23 +125,8 @@ func (w *Wrapper) PricingNodePrice(_ context.Context, req *protos.PricingNodePri
 	}
 	reqNode := req.GetNode()
 
-	var reqStartTime *metav1.Time
-	if startTimestamp := req.GetStartTimestamp(); startTimestamp != nil {
```

Comment:
keep the nil checks so we only set this if GetStartTimestamp()/GetEndTimestamp() were actually provided